sway-book/src/documentation/language/program-types/predicate.md

```diff
@@ -0,0 +1,17 @@
+# Predicates
+
+A predicate is an executable that represents a UTXO spending condition, such as a multisig predicate, which has restrictions on the VM instructions that can be used (e.g. no jumps).
+
+It does not need to be deployed to a blockchain because it only exists during a transaction. That being said, the predicate hash (specifically the root) is on-chain in the UTXO set.
+
```

Comment:
maybe "is on chain as the owner of one or more UTXOs"  is more precise. Also, it is the predicate _root_ rather than the hash.